default.nix

```diff
@@ -0,0 +1,104 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, fetchurl
+, runtimeShell
+, php82, php82Packages
+, ffmpeg_5-headless, mp4v2, fdk_aac, fdk-aac-encoder
+, useLibfdkFfmpeg ? false
+}:
+
+let
+  m4bToolPhp = php82.buildEnv {
+    extensions = ({ enabled, all }: enabled ++ (with all; [
+      dom mbstring tokenizer xmlwriter openssl
+    ]));
+
+    extraConfig = ''
+      date.timezone = UTC
+      error_reporting = E_ALL & ~E_STRICT & ~E_NOTICE & ~E_DEPRECATED
+    '';
+  };
+
+  m4bToolPhpPackages = php82Packages;
+
+  m4bToolComposer = pkgs.callPackage ./composer.nix {
+    php = m4bToolPhp;
+    phpPackages = m4bToolPhpPackages;
+  };
+
+  m4bToolFfmpeg = if useLibfdkFfmpeg then ffmpeg_5-headless.overrideAttrs (prev: rec {
+    configureFlags = prev.configureFlags ++ [
+      "--enable-libfdk-aac"
+      "--enable-nonfree"
+    ];
+    buildInputs = prev.buildInputs ++ [
+      fdk_aac
+    ];
+  }) else ffmpeg_5-headless;
+in
+m4bToolComposer.overrideAttrs (prev: rec {
+  pname = "m4b-tool";
+  version = "0.5";
```

Comment:
Wasn't sure what to call this since it's pulling from master.

Reply:
And as far as the tag goes, I would make the tag names immutable (0.5-prerelease-$DATE) or something would work. Right now I'm locking it to a particular commit which is less than ideal.